tools/remote_l4_npu/remote_l4_npu_smoke.py

```
@@ -0,0 +1,282 @@
+
+
+def _parse_args() -> argparse.Namespace:
+    parser = argparse.ArgumentParser(description=__doc__)
+    parser.add_argument("--machine-a", required=True, help="machine A daemon endpoint, HOST:PORT")
+    parser.add_argument("--machine-b", required=True, help="machine B daemon endpoint, HOST:PORT")
+    parser.add_argument("--machine-a-devices", default="0,1")
+    parser.add_argument("--machine-b-devices", default="0,1")
+    parser.add_argument("--platform", default="a2a3")
+    parser.add_argument("--runtime", default="tensormap_and_ringbuffer")
+    parser.add_argument("--session-timeout", type=float, default=120.0)
+    parser.add_argument("--session-listen-host", default="0.0.0.0")
```

Ruff (0.16.0)

[error] Possible binding to all interfaces

(S104)